.github/workflows/markdownlint.yml

```diff
@@ -0,0 +1,19 @@
+name: markdownlint
+
+on:
+  pull_request:
+    branches: [ master ]
```

Comment:
Eventually this will have a path filter https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths.